docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -22,16 +22,20 @@ Description. Include an overview of the feature and problem it solves, and where
 
 ## Authentication and authorization
 
-### Role-based access control is always enabled
+### No longer possible to disable Role-based access control
 
-[Role-based access control (RBAC)](({{< relref "../administration/roles-and-permissions/access-control/" >}})) is now enabled by default and you can't disable it using configuration options in Grafana.
+_Generally available in all editions of Grafana._
```

Comment:
Now that you've explained this a bit further, I'm not sure it makes sense to talk about availability like it's a feature when it's more of a fix. I want to run this by another colleague who's on PTO currently. They'll be back next week but in the mean time I'll review the rest of this and you can merge this with the main What's New PR. If I need to make an edit, I'll do so next week.